osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -97,11 +97,11 @@ public void Import(params string[] paths)
 
                 try
                 {
-                    notification.Text = $"Importing ({i} of {paths.Length})\n{Path.GetFileName(path)}";
+                    notification.Text = $"Importing ({++current} of {paths.Length})\n{Path.GetFileName(path)}";
                     using (ArchiveReader reader = getReaderFrom(path))
                         imported.Add(Import(reader));
 
-                    notification.Progress = (float)++i / paths.Length;
+                    notification.Progress = (float)(current - 1) / paths.Length;

```

Comment:
Why `current - 1`? This means after importing the first beatmap, the progress is going to be 0. And if there is only one beatmap to import, the progress is always going to be 0?

Reply:
current is incremented above, right?

Reply:
Yeah but imagine if `paths.Length == 1`, then we'd get to this line with `current = 1`, so `progress` would be `0 / paths.Length` `= 0`. Whereas previously it would do `1 / paths.Length`.

Reply:
What, skipping one of the ticks completely? Since when? I'd much rather advocate for having a debounce from completing notifications so that the progress bar fills up completely before firing up a second notification.

Reply:
Is this visible to you? I couldn't see it in testing.